### PR TITLE
Translation implementation

### PR DESCRIPTION
**Pull request by flokixdev**
Implementation is done to be framework agnostic, so e.g. contributte/translation is not used (we don't need Nette specific tools such as template translations, etc.)

### CODE REVIEW

File: src/Search.php [hunk 1]
```diff
@@ -22,22 +23,41 @@
...
+use Symfony\Component\Translation\Translator;
 use Tracy\Debugger;
 
 class Search
 {
 
-	/** @var Engine */
+	/**
+	 * @var Engine
+	 */
```
Comment (janbarasek) [inline src/Search.php]: @miloshavlicek codestyle should be `/** @var Engine */`.

File: src/Translation/TranslatorHelper.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	/**
+	 * If no translation is available, than this languages are used instead.
+	 *
+	 * @var string[]
+	 */
+	private $fallbackLanguages = ['en_US', 'cs_CZ'];
+
+
+	/**
+	 * TranslatorHelper constructor.
```
Comment (janbarasek) [inline src/Translation/TranslatorHelper.php]: Redundant comment

File: src/Translation/TranslatorHelper.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	 * @param Translator $translator
+	 */
+	public function __construct(
+		Translator $translator
+	)
```
Comment (janbarasek) [inline src/Translation/TranslatorHelper.php]: Codestyle should be `public function __construct(Translator $translator)`

File: src/Search.php (hunk 1 above)
Comment (flokixdev) [reply src/Search.php]: I know, that's written in the log, however I am not able to fix it automatically on localhost
`php temp/code-checker/code-checker --short-arrays --strict-types --fix --no-progress` and
`php temp/coding-standard/ecs check src --config temp/coding-standard/coding-standard-php71.yml` does not change anything and results as done ok.

Comment (janbarasek) [reply src/Search.php]: Ok I understand. And who has to fix the codestyle? I?

File: src/Translation/TranslatorHelper.php (hunk 3 above)
Comment (flokixdev) [reply src/Translation/TranslatorHelper.php]: That's really pain since there is no Codestyle spec for IDE.

File: src/Search.php (hunk 1 above)
Comment (flokixdev) [reply src/Search.php]: I'll fix it. Give me a minute.

File: src/Translation/TranslatorHelper.php (hunk 3 above)
Comment (janbarasek) [reply src/Translation/TranslatorHelper.php]: I don't think I can accept code that breaks tests no matter how it originated.

Comment (flokixdev) [reply src/Translation/TranslatorHelper.php]: I don't say you should accept it. Anytime I will send a PR and some tests fails, I'll try to fix it, sure. I just say that codestyle is in this case a pain.

**Comment by janbarasek** [comment]
Thanks!

**Pull request merged**